src/Umbraco.Web/PropertyEditors/ValueConverters/MediaPickerWithCropsValueConverter.cs

```diff
@@ -24,96 +22,58 @@ public MediaPickerWithCropsValueConverter(IPublishedSnapshotAccessor publishedSn
 
         public override PropertyCacheLevel GetPropertyCacheLevel(IPublishedPropertyType propertyType) => PropertyCacheLevel.Snapshot;
 
-        /// <summary>
-        /// Enusre this property value convertor is for the New Media Picker with Crops aka MediaPicker 3
-        /// </summary>
         public override bool IsConverter(IPublishedPropertyType propertyType) => propertyType.EditorAlias.Equals(Core.Constants.PropertyEditors.Aliases.MediaPicker3);
 
-        /// <summary>
-        /// Check if the raw JSON value is not an empty array
-        /// </summary>
-        public override bool? IsValue(object value, PropertyValueLevel level) => value?.ToString() != "[]";
+        public override bool? IsValue(object value, PropertyValueLevel level)
+            => value?.ToString() is string stringValue &&
+                stringValue != null &&
```

Comment:
```suggestion
```

This is getting checked for `null` again on the next line! 😅 and since it `is string` it can not be `null` anwyay.